### DISCUSSION TITLE
Openzeppelin contracts Unlicenced

### DISCUSSION THREAD

**Discussion by taofeeq125** [Cyfrin/security-and-auditing-full-course-s23#89] [Q&A]
My Openzeppelin is unlicensed what is the way out

**Comment by johnsonstephan** [comment]
Hi @taofeeq125 - Kindly share (1) specifics on the issue (2) what your intended result is (3) steps leading up that produced the error.

See more [here](https://stackoverflow.com/help/how-to-ask) on formatting your question so we can help you resolve.

**Reply by cromewar** [reply]
Please follow up on the instructions @johnsonstephan mentioned so we can provide better assistance @taofeeq125.
